node/node_newblock.go

```diff
@@ -92,7 +92,7 @@ func (node *Node) WaitForConsensusReadyv2(readySignal chan struct{}, stopChan ch
 							ctxerror.New("cannot commit new block").
 								WithCause(err))
 						continue
-					} else if err := node.proposeShardState(newBlock); err != nil {
+					} else if err := node.proposeShardStateWithoutBeaconSync(newBlock); err != nil {
```

Comment:
We need to keep the overall beaconchain-initiated network resharding as discussed yesterday.  Please revert this.

Reply:
this is for the new blskey join, which is a temp fix. we must use this to bypass the current blockers.